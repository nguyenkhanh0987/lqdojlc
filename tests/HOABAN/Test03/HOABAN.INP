3 10 12345
1 1
2 1
3 1
3 1
2 1
1 1
1 1
1 1
1 1
1 1