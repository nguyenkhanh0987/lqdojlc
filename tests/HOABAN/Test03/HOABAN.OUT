12352 12347 12346 